docs/3_usage.rst

```diff
@@ -107,3 +107,24 @@ In your code, you can use the ``axes.utils.reset`` function.
    Please note that if you give both ``username`` and ``ip`` arguments to ``reset``
    that attempts that have both the set IP and username are reset.
    The effective behaviour of ``reset`` is to ``and`` the terms instead of ``or`` ing them.
+
+
+
+Data privacy and GDPR
+^^^^^^^^^^^^^^^^^^^^^
+
+Most European countries have quite strict laws regarding data protection and privacy. It's highly recommended and good 
+practice to treat your sensitive user data with care. The general rule here is that you shouldn't store what you don't need.
+
+When dealing with brute-force protection, the IP address and the username (often the email address) are most crucial.
+Given that you can perfectly use `django-axes` without locking the user out by IP but by username, it does make sense to
+avoid storing the IP address at all. You can not lose what you don't have.
```

Comment:
Afaik IP address is not sensitive user data or personal information per-se, if it's not combined with other identifiable data that points to a single user. And, on the other hand, we are storing login attempts which can be made by anyone on the world wide web. What data needs to be stored in what scenarios is a pretty interesting question that would be worth discussing a bit more about, but I'm not an expert on this field, so maybe it's easiest to let people open an issue ticket or a PR if they seem room for improvement on the instructions of use or any other points :)

Reply:
Great idea! And thx for your effort! 👌